interfaces/builtin/common_test.go

```diff
@@ -84,6 +85,15 @@ func MockEvalSymlinks(test *testutil.BaseTest, fn func(string) (string, error))
 	})
 }
 
+// MockReadDir replaces the io/ioutil.ReadDir function used inside the caps package.
+func MockReadDir(test *testutil.BaseTest, fn func(string) ([]os.FileInfo, error)) {
```

Comment:
Typically we return a function that restores the mocking operation. I don't oppose this but we can change that on a drive-by next time.

Reply:
I was just following what we did for MockEvalSymlinks, directly above.